torch/csrc/Storage.cpp

```diff
@@ -234,9 +236,10 @@ static PyObject* THPStorage_get(THPStorage* self, PyObject* index) {
     }
 
     const auto& storage = THPStorage_Unpack(self);
-    uint8_t* data = storage.data<uint8_t>();
-
     at::StorageImpl* old_storage_impl = storage.unsafeGetStorageImpl();
+
+    uint8_t* data = old_storage_impl->mutable_unsafe_data<uint8_t>();
```

Comment:
I think this change is ok, but I don't like diffs where something safe goes to unsafe. Needs more explanation